code/modules/power/supermatter/supermatter.dm

```diff
@@ -348,7 +348,7 @@ GLOBAL_DATUM(main_supermatter_engine, /obj/machinery/power/supermatter_crystal)
 		set_delam(SM_DELAM_PRIO_NONE, SM_DELAM_STRATEGY_PURGE) // This one cant clear any forced delams.
 	delamination_strategy.delam_progress(src)
 	// BUBBER EDIT ADDITION BEGIN - DELAM_SCRAM
-	if(damage > SUPERMATTER_SUPPRESSION_THRESHOLD && is_main_engine && !suppression_fired)
+	if(damage > SUPERMATTER_SUPPRESSION_THRESHOLD && is_main_engine && !suppression_fired && world.time - SSticker.round_start_time < 30 MINUTES)
```

Comment:
Since this time (30 minutes) shows up in a few places in the code now, maybe it could be a define? Not mandatory tho

Reply:
Yeah good idea